test/verify/check-system-info

```diff
@@ -911,6 +911,17 @@ password=foobar
         profile = m.execute(cmd + " --show").strip()
         b.wait_text("#crypto-policy-button", shown_profile_text(profile))
 
+        # RHEL 8 has no SHA1 policy, so do not show it.
+        b.click("#crypto-policy-button")
+        func = b.wait_not_present if m.image.startswith('rhel-8') or m.image.startswith('centos-8') else b.wait_visible
+        func(".pf-c-menu__item-main .pf-c-menu__item-text:contains('Default:sha1')")
```

Comment:
That's a funky way to write this, instead of an if/else and a constant for the selector, but sure :grimacing: